core/src/main/java/com/datastrato/gravitino/storage/relational/po/TopicPO.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.storage.relational.po;
+
+import com.google.common.base.Preconditions;
+import java.util.Objects;
+import lombok.Getter;
+
+@Getter
+public class TopicPO {
+  private Long topicId;
+  private String topicName;
+  private Long metalakeId;
+  private Long catalogId;
+  private Long schemaId;
+  private String comment;
+  private String properties;
+  private String auditInfo;
+  private Long currentVersion;
+  private Long lastVersion;
+  private Long deletedAt;
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof TopicPO)) {
+      return false;
+    }
+    TopicPO topicPO = (TopicPO) o;
+    return Objects.equals(topicId, topicPO.topicId)
+        && Objects.equals(topicName, topicPO.topicName)
+        && Objects.equals(metalakeId, topicPO.metalakeId)
+        && Objects.equals(catalogId, topicPO.catalogId)
+        && Objects.equals(schemaId, topicPO.schemaId)
+        && Objects.equals(auditInfo, topicPO.auditInfo)
```

Comment:
It seems that the `properties` field is not added to the equals / hashCode computing.